src/index.js

```diff
@@ -98,16 +98,22 @@ class FlipMotion extends Component<void, Props, State> {
     }
   }
   render() {
-    const { style, childStyle } = this.props
+    const style = this.props.style
+    const childStyle = this.props.childStyle
+    const Element = this.props.element || "div"
+    const ChildElement = this.props.childElement || "div"
```

Comment:
Agreed. I want to be explicit that the constants here are cased as `Component` and `ChildComponent` because React renders components written in camelCase as actual HTML elements (i.e. invalid `<childComponent>` elements in HTML), which might just be a JSX quirk.